app/services/get_into_teaching_api/constants.rb

```diff
@@ -1,5 +1,12 @@
 module GetIntoTeachingApi
   module Constants
+    EVENT_TYPES =
+      {
+        "Train to Teach Event" => 222_750_001,
+        "School and University Event" => 0o000000000,
```

Comment:
is that correct?

Reply:
It's just dummy data for now; the event types are still up in the air so we will need to revisit this once they've been worked out...